cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/SystemAppTask.java

```diff
@@ -114,6 +115,7 @@ private Injector createInjector() {
       new MessagingClientModule(),
       new LocalLocationModule(),
       new SecureStoreClientModule(),
+      new AuthenticationContextModules().getProgramContainerModule(cConf),
```

Comment:
For system apps, we should probably be using getMasterModule() instead

Reply:
Done